src/bank01_player.asm

```diff
@@ -671,20 +671,16 @@ showPlayerAtTile:
     pop  HL                                            ;; 01:44d6 $e1
     ret                                                ;; 01:44d7 $c9
 
-; A = direction: 1=west, 2=east, 4=south, 8=north
+; A = direction: 1=west, 2=east, 4=south, 8=north (flip of traditional directions)
+; D = scroll speed
 scrollRoom:
-    ld   E, A                                          ;; 01:44d8 $5f
```

Comment:
Silly request, but assuming this is still at `01:44d8` (I just checked and it is) leave it unmodified for easy reference to the position.

Reply:
Done